### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#54297]
#### VFS team name
Sitewide Facilities

#### VFS product name
VAMC

#### VFS feature name
VAMC Lovell Federal Health care

#### Point of Contact/Reviewers
Allison Christman - @allison0034 - Design

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
Some of the accordions have an expand all between each accordion. 

### Link, screenshot or steps to recreate
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/52672_allison_040702.png

### VA.gov Experience Standard
[Category Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
There should be only 1 expand all per accordion set, remove the ones between each accordion.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by davidmpickett** [comment]
Removed Lovell label as this is true on all VAMC pages

**Comment by laflannery** [comment]
Duplicate of [#12807](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12087)
